test/helper.lua

```diff
@@ -230,4 +237,77 @@ function helpers.get_other_storage_bucket_id(cluster, bucket_id)
     ]], {bucket_id})
 end
 
+-- Determine a replicaset alias by a server object.
+function helpers.get_replicaset_alias(cluster, needle_server)
+    for _, replicaset in ipairs(cluster.replicasets) do
+        for _, server in ipairs(replicaset.servers) do
+            if server.alias == needle_server.alias then
+                return replicaset.alias
+            end
+        end
+    end
+    error(('Unable to find replicaset alias for server %s'):format(
+        needle_server.alias))
+end
+
+-- Accumulate statistics from given storages.
+--
+-- The statistics is grouped by replicasets.
+--
+-- Example of a return value:
+--
+-- {
+--     ['s-1'] = {
+--         select_requests = 1,
+--     },
+--     ['s-2'] = {
+--         select_requests = 0,
+--     },
+-- }
+function helpers.collect_storage_stat(cluster, aliases)
```

Comment:
I would use `checks()` here.

Reply:
Now I'm going to drop the 'aliases' parameter, because we can determine storage nodes automatically by presence of the 'crud-storage' role. I can check that 'cluster' is a table, okay.